cms/djangoapps/contentstore/management/commands/export_content_library.py

```diff
@@ -0,0 +1,64 @@
+"""
+Script for exporting a content library from Mongo to a tar.gz file
+"""
+from __future__ import print_function
+import os
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locator import LibraryLocator
+from xmodule.modulestore.django import modulestore
+
+from cms.djangoapps.contentstore import tasks
+
+
+class Command(BaseCommand):
+    """
+    Export the specified content library into a directory.  Output will need to be tar zxcf'ed.
+    """
+    help = 'Export the specified content library into a directory'
+
+    def add_arguments(self, parser):
+        parser.add_argument('library_id')
+        parser.add_argument('output_path', nargs='?')
+
+    def handle(self, *args, **options):
+        """
+        Given a content library id, and an output_path folder.  Export the
+        corresponding course from mongo and put it directly in the folder.
+        """
+        module_store = modulestore()
+        try:
+            library_key = CourseKey.from_string(options['library_id'])
+            assert isinstance(library_key, LibraryLocator)
+        except InvalidKeyError:
+            raise CommandError(u'Invalid library ID: "{0}".'.format(options['library_id']))
+        except AssertionError:
```

Comment:
I don't like using an assert statement and catching the assert as a way of testing a condition. Can we change this to an if-statement instead?

Reply:
Done, changed to an if-statement.